roles/kubernetes-apps/local_volume_provisioner/templates/configmap.yml.j2

```diff
@@ -7,7 +7,8 @@ metadata:
   name: local-volume-config
   namespace: {{ system_namespace }}
 data:
-  storageClassMap: |
-    local-storage:
-      hostDir: "{{ local_volume_base_dir }}"
-      mountDir: "/mnt/local-storage/"
+  "local-storage": |
```

Comment:
Do you a pointer to a documentation for this change?

Reply:
Here: https://github.com/kubernetes-incubator/external-storage/blob/master/local-volume/provisioner/CHANGELOG.md

Reply:
> Configmap data is changed from json to yaml syntax.

I think the previous form was correct. Also looks like the key `storageClassMap` is still needed too

Reply:
Ha @mirwan , I just fixed this myself today and was about to submit a PR. Could you please template the "local-storage" as well? It would look like this:
```
data: 
  {{ local_volume_storage_class }}: |
    {"hostDir": "{{ local_volume_base_dir }}", "mountDir": "{{ local_volume_mount_dir }}"}
```

And of course adding that variable in main.yaml. That would really help out my use case.

Reply:
Done @eyeofthefrog

Reply:
Thanks!